benchmarks/web-serving/faban_client/Dockerfile

```diff
@@ -9,9 +9,8 @@ RUN apt-get update && apt-get install -y \
 
 # Setup Faban 
 
-RUN wget http://faban.org/downloads/faban-kit-latest.tar.gz
-RUN tar zxvf faban-kit-latest.tar.gz
-
+RUN wget http://faban.org/downloads/faban-kit-1.4.tar.gz
```

Comment:
should we add faban version as an environment variable like we do in case of web-search client?

Reply:
Yes please do it like in web-search.
also, fix the java version.

Reply:
I think it is covered in #143

Reply:
@altostratous can you please fix the java version, so it won't break in future.

Reply:
Yes, I fixed it.

Reply:
Yes, I fixed it.